### ISSUE TITLE
`require-meta-schema`: False positive 

### ISSUE THREAD

**Issue by fisker** [not-an-aardvark/eslint-plugin-eslint-plugin#110]
When schema is a reference, `require-meta-schema` reported on following case.

```js
const schema = [
	{
		type: 'object',
		properties: {
			sortCharacterClasses: {
				type: 'boolean',
				default: true
			}
		}
	}
];

module.exports = {
	create,
	meta: {
		type: 'suggestion',
		docs: {
			description: 'Improve regexes by making them shorter, consistent, and safer.',
			url: getDocumentationUrl(__filename)
		},
		fixable: 'code',
		schema,
		messages
	}
};
```

I'll look into it.
